release-calendar/src/app.ts

```diff
@@ -13,9 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import {createConnection, Connection} from 'typeorm';
-import {Release} from './entities/release';
+import 'reflect-metadata';
+import {createConnection} from 'typeorm';
+import {Release, Channel} from './entities/release';
```

Comment:
Can we move `Channel` into a type def file in `src/types`? That way we don't have to import it in every file